frontend/src/component/impact-metrics/ImpactMetrics.tsx

```diff
@@ -100,33 +103,49 @@ export const ImpactMetrics: FC = () => {
         [deleteChart],
     );
 
-    const gridItems: GridItem[] = useMemo(
-        () =>
-            charts.map((config, index) => {
-                const existingLayout = layout?.find(
-                    (item) => item.i === config.id,
-                );
-                return {
-                    id: config.id,
-                    component: (
-                        <ChartItem
-                            config={config}
-                            onEdit={handleEditChart}
-                            onDelete={handleDeleteChart}
-                        />
-                    ),
-                    w: existingLayout?.w ?? 6,
-                    h: existingLayout?.h ?? 4,
-                    x: existingLayout?.x,
-                    y: existingLayout?.y,
-                    minW: 4,
-                    minH: 2,
-                    maxW: 12,
-                    maxH: 8,
-                };
-            }),
-        [charts, layout, handleEditChart, handleDeleteChart],
-    );
+    const gridItems: GridItem[] = useMemo(() => {
```

Comment:
I am appending the plausible item to the existing chartItems list so it will get same layout.